app/scripts/modules/core/src/modal/wizard/WizardModal.tsx

```diff
@@ -171,7 +178,7 @@ export class WizardModal<T = {}> extends React.Component<IWizardModalProps<T>, I
               <Modal.Header>{heading && <h3>{heading}</h3>}</Modal.Header>
 
               <Modal.Body>
-                {loading ? (
+                {loading && initialized ? (
```

Comment:
should this be `loading || !initialized` ?  maybe it doesn't make any difference.

Reply:
👍